po/it.po

```diff
@@ -0,0 +1,176 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the timetable package.
+# ALBANO BATTISTELLA <albano_battistella@>, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: timetable\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-11-04 13:12+0100\n"
+"PO-Revision-Date: 2020-07-26 13:00+0100\n"
+"Last-Translator: Albano_battistella <Albano_battistella@hotmail.com>\n"
+"Language-Team: IT <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/io.github.kaschpal.timetable.desktop.in:3 src/MainWindow.py:38
+#: src/MainWindow.py:262 src/MainWindow.py:316
+msgid "timetable"
+msgstr "timetable"
+
```

Comment:
Thanks for contributing! I want the app behave as GNOMEish as possible, could you also translate the name of the program? If in doubt, in the de.po, there's "Stundenplan" for timetable/orario.

Reply:
Hi!
Timetable is good for me.

Reply:
Then we keep it.